Documentation/gettingstarted/servicemesh/envoy-traffic-management.rst

```diff
@@ -69,17 +61,17 @@ traffic from the ``client2`` pod:
 
 .. code-block:: shell-session
 
-    $ cilium hubble port-forward &
-    $ hubble observe --from-pod cilium-test/$CLIENT2 -f
+    $ kubectl -n kube-system port-forward deployment/hubble-relay 4245:4245 &
```

Comment:
The pre-reqs do not include enabling hubble-relay, perhaps the `helm upgrade` command can include enabling hubble-relay?

Reply:
good point :100: